### ISSUE TITLE
When declining or merging, no message is sent to a channel about that event

### ISSUE THREAD

**Issue by aaronrothschild** [mattermost-community/mattermost-plugin-bitbucket#10]
I'm not sure what the Bitbucket equivalent is "closed" status on a PR. But neither decline or merge deliver an event

**Comment by kosgrz** [comment]
@DHaussermann, I agree that some events are very noisy and I will fix it, but when you merge a PR and you have the `pull_reviews` feature enabled, you should see a notification like this:
![image](https://user-images.githubusercontent.com/45372453/105621836-0ca9b880-5e0c-11eb-9230-2890ef0e4e00.png)
You didn't see anything like this? Similar message is sent when a PR is declined.
